docs/contributing/README.md

```diff
@@ -15,7 +15,7 @@ Our documentation is run by serving the docs folder at a given port.
 From the root folder on a command console, enter the command:
 
 ```sh
-npm run start:docs
+yarn run start:docs
```

Comment:
```suggestion
yarn start:docs
```

Reply:
Thanks for the review, Simone. I'm used to running `yarn run` instead of  `yarn` to execute a command, if you prefer the version without, I'll update the doc.

Reply:
Done